### PR TITLE
Disable static analyzer for message semaphore creation

### PR DESCRIPTION
**Pull request by sergiocampama**

### CODE REVIEW

**Comment by sergiocampama** [comment]
@thomasvl fyi
https://github.com/google/protobuf/issues/2627

File: objectivec/GPBMessage_PackagePrivate.h [hunk 1]
```diff
@@ -108,9 +108,17 @@ CF_EXTERN_C_BEGIN
 NS_INLINE void GPBPrepareReadOnlySemaphore(GPBMessage *self) {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdirect-ivar-access"
+
+// Starting on Xcode 8.3, the static analyzer complains that the dispatch_once_t variable
```
Comment (thomasvl) [inline objectivec/GPBMessage_PackagePrivate.h]: wrap to 80c like the majority of the file.

Comment (sergiocampama) [inline objectivec/GPBMessage_PackagePrivate.h]: Done.

**Comment by thomasvl** [comment]
fyi - travis doesn't have results from the run, so I told it to rerun that, hopefully we'll get results this time.

**Pull request merged**